test/common/upstream/cds_api_impl_test.cc

```diff
@@ -86,9 +85,10 @@ class CdsApiImplTest : public testing::Test {
   Stats::IsolatedStoreImpl store_;
   Http::MockAsyncClientRequest request_;
   CdsApiPtr cds_;
-  Event::MockTimer* interval_timer_{new Event::MockTimer(&dispatcher_)};
```

Comment:
Creating the `MockTimer` sets up some `EXPECT_CALL` which doesn't apply in the `CdsApiImplTest.InvalidOptions` case. I think we maybe reject bad configurations earlier than this call now.